packages/cmrjs/src/cmr-utils.js

```diff
@@ -44,6 +44,14 @@ function getS3UrlOfFile(file) {
   throw new Error(`Unable to determine location of file: ${JSON.stringify(file)}`);
 }
 
+function getFilename(file) {
+  if (file.name) return file.name;
+  if (file.filename) return path.basename(file.filename);
+  if (file.filepath) return path.basename(file.filepath);
+  if (file.key) return path.basename(file.key);
+  throw new Error(`Unable to determine name of file: ${JSON.stringify(file)}`);
```

Comment:
Should we throw an error here? I'm not sure if is beneficial and may only cause unexpected failures

Reply:
I don't have a problem with it returning `undefined` in this case, and let the calling code handle that.